run.sh

```diff
@@ -19,6 +22,8 @@ sed -i "s|MAKEFLAGS=.*|MAKEFLAGS=-j$(nproc)|" /etc/makepkg.conf
 useradd -m user
 cd /home/user
 
+# Copy PKGBUILD and *.install scripts
+cp "$PKGBUILD_DIR/*install" ./
```

Comment:
This doesn't work. `*` will not get expanded. Please change to `"PKGBUILD_DIR"/*.install`.

Reply:
Oh, of course. Sorry. 
Changed.